### PR TITLE
First approach to range-number validator

### PR DESCRIPTION
**Pull request by v-borrego**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -9,5 +9,6 @@ module.exports = {
   rules: {
     '@typescript-eslint/no-explicit-any': 'off',
     '@typescript-eslint/explicit-function-return-type': 'off',
+    '@typescript-eslint/no-use-before-define': 'off',
```
Comment (nasdan) [inline .eslintrc.js]: Finally, we don't need to disabled this rule, isn't it?

File: src/validator.ts [hunk 2]
```diff
@@ -1,27 +1,49 @@
...
+    !isDefined(value) ||
+    (validateType(value) && validate(value, customArgs.min, customArgs.max));
 
   return {
     succeeded,
     message: succeeded
       ? ''
-      : buildCustomMessage((message as string) || defaultMessage, args),
+      : parseMessageWithCustomArgs(
+          (message as string) || defaultMessage,
```
Comment (nasdan) [inline src/validator.ts]: Why is it necessary the `|| defaultMessage` if we are using it on line 33?

File: .eslintrc.js (hunk 1 above)
Comment (v-borrego) [reply .eslintrc.js]: you're right

File: src/validator.ts (hunk 2 above)
Comment (v-borrego) [reply src/validator.ts]: It is not necessary. Updated.

File: examples/js/src/index.js [hunk 3]
```diff
@@ -9,17 +9,16 @@ const validationSchema = {
```
Comment (nasdan) [inline examples/js/src/index.js:23]: Pending to add customArgs in examples?

Comment (v-borrego) [reply examples/js/src/index.js:23]: Done!

**Pull request merged**